VideoLocker/res/layout/edx_spinner_item.xml

```diff
@@ -12,4 +12,5 @@
     android:singleLine="true"
     android:textColor="@color/edx_grayscale_neutral_dark"
     android:textSize="@dimen/edx_x_small"
+    android:textDirection="locale"
```

Comment:
Again, if we want to set the text directions to `locale`, then it should be done globally. I don't yet see any compelling reason for it though.